WikipediaUnitTests/Code/MWKTitleTests.m

```diff
@@ -48,7 +48,6 @@ - (void)testSimple {
     MWKTitle* title = [MWKTitle titleWithString:@"Simple" site:site];
     XCTAssertEqualObjects(title.text, @"Simple", @"Text form is full");
     XCTAssertNil(title.fragment, @"Fragment is nil");
-    XCTAssertEqualObjects(title.escapedFragment, @"", @"Fragment for URL is empty string");
```

Comment:
This is just fallout from removing `escapedFragment` entirely - it's not used anywhere in the code, only this test. https://github.com/wikimedia/wikipedia-ios/pull/706/commits/b05102e9ca86b3e093ac1935aa1b68cfa403f64d
and
https://github.com/wikimedia/wikipedia-ios/pull/706/commits/5d1eb0bca59f3088d07b94e8fdfa99ba8e9635fb
were just code pruning